.env.example

```diff
@@ -1,4 +1,3 @@
-DAPI_CLIENT_SEEDS=ip1,ip2
-DAPI_CLIENT_PORT=3000
+DAPI_SEED=ip:port
```

Comment:
I would keep it empty otherwise `dotenv-safe` won't work. You can just add a comment with format